CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ from `algosdk` and sends it to the network.
 
 #### @algo-builder/web
 - Added `appendSignMultisigTransaction` function to `WebMode` for appending signature to multisig transaction in the algosigner.
+- Added `MultiSignature` support in `executeTx` method.
```

Comment:
Lets add a bit more details here. Its only for the algosinger right?

Reply:
yes. I will add that.